### PR TITLE
Example LSTMCell

### PR DESCRIPTION
**Pull request by CarlosJose126**
Fixes #{51801}
LSTMCell example updated

### CODE REVIEW

File: torch/nn/modules/rnn.py [hunk 1]
```diff
@@ -1029,15 +1029,18 @@ class LSTMCell(RNNCellBase):
...
+        >>> rnn = nn.LSTMCell(10, 20)        # (input_size, hidden_size)
+        >>> input = torch.randn(2, 3, 10)    # (time_steps, batch, input_size)
+        >>> hx = torch.randn(3, 20)          # (batch, hidden_size)
+        >>> cx = torch.randn(3, 20)          
         >>> output = []
-        >>> for i in range(6):
+        >>> for i in range(input.size()[0]):  
                 hx, cx = rnn(input[i], (hx, cx))
                 output.append(hx)
+        >>> output = torch.stack(output, dim = 0)
```
Comment (zou3519) [inline torch/nn/modules/rnn.py:1043]: nit: `dim = 0` -> `dim=0`, Python linters will complain about spacing around the equals

File: torch/nn/modules/rnn.py [hunk 2]
```diff
@@ -1029,15 +1029,18 @@ class LSTMCell(RNNCellBase):
         where :math:`k = \frac{1}{\text{hidden\_size}}`
 
     Examples::
+    
+
```
Comment (zou3519) [inline torch/nn/modules/rnn.py:1033]: Do the spaces do anything? Could we see a screenshot of the new docs?

**Review by zou3519 — COMMENTED** [review]
Minor nit, but looks good to me otherwise. Could we also see a screenshot of the docs to check the rendering?

File: torch/nn/modules/rnn.py [hunk 3]
```diff
@@ -1028,16 +1028,17 @@ class LSTMCell(RNNCellBase):
         All the weights and biases are initialized from :math:`\mathcal{U}(-\sqrt{k}, \sqrt{k})`
         where :math:`k = \frac{1}{\text{hidden\_size}}`
 
-    Examples::
+    Examples::
```
Comment (zou3519) [inline torch/nn/modules/rnn.py:1031]: Please fix the trailing whitespace

File: torch/nn/modules/rnn.py [hunk 4]
```diff
@@ -1028,16 +1028,17 @@ class LSTMCell(RNNCellBase):
...
+    Examples::    
 
-        >>> rnn = nn.LSTMCell(10, 20)
-        >>> input = torch.randn(3, 10)
-        >>> hx = torch.randn(3, 20)
-        >>> cx = torch.randn(3, 20)
+        >>> rnn = nn.LSTMCell(10, 20)        # (input_size, hidden_size)
+        >>> input = torch.randn(2, 3, 10)    # (time_steps, batch, input_size)
+        >>> hx = torch.randn(3, 20)          # (batch, hidden_size)
+        >>> cx = torch.randn(3, 20)
```
Comment (zou3519) [inline torch/nn/modules/rnn.py:1036]: Trailing whitespace here as well

File: torch/nn/modules/rnn.py [hunk 5]
```diff
@@ -1028,16 +1028,17 @@ class LSTMCell(RNNCellBase):
...
-        >>> input = torch.randn(3, 10)
-        >>> hx = torch.randn(3, 20)
-        >>> cx = torch.randn(3, 20)
+        >>> rnn = nn.LSTMCell(10, 20)        # (input_size, hidden_size)
+        >>> input = torch.randn(2, 3, 10)    # (time_steps, batch, input_size)
+        >>> hx = torch.randn(3, 20)          # (batch, hidden_size)
+        >>> cx = torch.randn(3, 20)          
         >>> output = []
-        >>> for i in range(6):
+        >>> for i in range(input.size()[0]):
```
Comment (zou3519) [inline torch/nn/modules/rnn.py:1038]: And here

Comment (CarlosJose126) [reply torch/nn/modules/rnn.py]: done! Thank you for your patience :) 
please let me know if I should correct anything else

File: torch/nn/modules/rnn.py [hunk 6]
```diff
@@ -1029,15 +1029,16 @@ class LSTMCell(RNNCellBase):
         where :math:`k = \frac{1}{\text{hidden\_size}}`
 
     Examples::
-
-        >>> rnn = nn.LSTMCell(10, 20)
-        >>> input = torch.randn(3, 10)
-        >>> hx = torch.randn(3, 20)
+
```
Comment (zou3519) [inline torch/nn/modules/rnn.py:1032]: One last trailing whitespace fix here and then we should be good!

**Review by zou3519 — APPROVED** [review]
I have no clue what's up with the whitespace, but it should be harmless:
image

Going to merge this unless you want to try to fix it more

**Comment by zou3519** [comment]
@CarlosJose126 could you rebase this PR on top of master? There appear to be merge conflicts

**Comment by CarlosJose126** [comment]
> @CarlosJose126 could you rebase this PR on top of the master? There appear to be merge conflicts

I tried to do that, but it didn't work. Anyway,  if you could do the example update directly without using my PR or creating a new PR it would be ok for me. The important thing is to have the documentation updated for the community :)

**Comment by zou3519** [comment]
> I tried to do that, but it didn't work. Anyway, if you could do the example update directly without using my PR or creating a new PR it would be ok for me. The important thing is to have the documentation updated for the community :)

I can force push to your branch, let me try that...

**Comment by CarlosJose126** [comment]
Hey @ zou3519 thanks for your help with the PR. I took a look at the LSTMCell documentation at https://pytorch.org/docs/stable/generated/torch.nn.LSTMCell.html and it hasn't been updated yet. This is normal?

**Comment by zou3519** [comment]
@CarlosJose126 sorry for the late reply. We merged this too late for the 1.8 release, but the change is on master (and therefore will be a part of the 1.9 release: https://pytorch.org/docs/master/generated/torch.nn.LSTMCell.html?highlight=lstmcell#torch.nn.LSTMCell).

I'll try to get this change cherry-picked into the 1.8.1 release so that it can come sooner. Thanks for pointing it out!
